CHANGES.rst

```diff
@@ -1652,6 +1652,10 @@ Bug Fixes
 
   - Fixed bug in Ci definition. [#5106]
 
+  - Non-ascii cds unit strings are now correctly represented using ``str`` also
+    on python2. This solves bugs in parsing coordinates involving strings too.
+    [#5355]
```

Comment:
FWIW, it references the PR, and the PR references the issue, and that's enough for me!